From bf1727f2eb3778c35d8a742f7d88866fd9a9cb55 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Wed, 6 Mar 2013 15:27:58 +0100
Subject: [PATCH] patch: strict-overflow-warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Squashed commit of the following:

commit f059fde47f4ae7d642d9a256b9b07297ad9aabd2
Author: Nils Philippsen <nils@redhat.com>
Date:   Wed Mar 6 12:16:43 2013 +0100

    fix bogus overflow warning

    tile-swap.c: In function ‘tile_swap_command’:
    tile-swap.c:721:6: warning: assuming signed overflow does not occur when
    assuming that (X + c) < X is always false [-Wstrict-overflow]
    (cherry picked from commit bc610c8fbd61e9574a2c378829b6ab91047d61b7)
---
 app/base/tile-swap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/base/tile-swap.c b/app/base/tile-swap.c
index b65526e..ecc7dc1 100644
--- a/app/base/tile-swap.c
+++ b/app/base/tile-swap.c
@@ -110,7 +110,7 @@ static void          tile_swap_gap_destroy    (SwapFileGap *gap);
 
 static SwapFile     * gimp_swap_file   = NULL;
 
-static const gint64   swap_file_grow   = 1024 * TILE_WIDTH * TILE_HEIGHT * 4;
+static const guint64  swap_file_grow   = 1024 * TILE_WIDTH * TILE_HEIGHT * 4;
 
 static gboolean       seek_err_msg     = TRUE;
 static gboolean       read_err_msg     = TRUE;
-- 
1.8.1.4

